### PR TITLE
[ BUGS-5985 ] - Dynamic class properties cause deprecation notices in php 8.2

### PR DESCRIPTION
**Pull request by stovak**
Dynamic class properties are deprecated in php 8.2

### CODE REVIEW

File: src/Commands/MysqlSlowCommand.php [hunk 1]
```diff
@@ -160,7 +172,7 @@ private function ParseMysqlSlowLog($dir, $options)
      */
     private function DefineSiteEnv($site_env)
     {
-        list($this->site, $this->environment) = $this->getSiteEnv($site_env);
+        [$this->site, $this->environment] = $this->getSiteEnv($site_env);
```
Comment (greg-1-anderson) [inline src/Commands/MysqlSlowCommand.php:175]: TIL this works all the way back to PHP 7.1.

**Review by greg-1-anderson — APPROVED** [review]

File: src/Commands/MysqlSlowCommand.php (hunk 1 above)
Comment (namespacebrian) [reply src/Commands/MysqlSlowCommand.php:175]: TIL multiple return values works in PHP at all

**Comment by namespacebrian** [comment]
Looks like you've covered more examples than were explicitly documented in the ticket, which is great.  Did you use a code analysis tool to identify them?

**Review by namespacebrian — APPROVED** [review]

**Comment by stovak** [comment]
@namespacebrian I used copilot, opened all the classes, and any class init that used the logger, I added the logger var. Then did a phpstorm -> reformat which changed the assignment structure.

**Pull request merged**